comictaggerlib/taggerwindow.py

```diff
@@ -1203,24 +1205,25 @@ def commit_metadata(self) -> None:
                 self.update_menus()
             self.fileSelectionList.update_current_row()
 
-            self.metadata = self.comic_archive.read_metadata(self.load_data_style)
+            self.metadata, success = self.overlay_ca_read_style(self.load_data_styles, self.comic_archive)
```

Comment:
`success` is never used, either remove the return value from `overlay_ca_read_style` or do something with it here